release-content/0.14/release-notes/13417_implement_the_full_set_of_sort_methods_on_QueryIter.md

```diff
@@ -1 +1,114 @@
-TODO
+Currently, query iteration order is not guaranteed. If we wish to work with our query items in a certain order, we need sorting.
+We might want sort our queries to f.e. display a list of things, or fold over our query with some order-dependent math operation.
+In 0.13 a sort could look like this:
+
+```rust
+#[derive(Component, Copy, Clone, Deref)]
+pub struct Attack(pub usize)
+
+fn handle_enemies(enemies: Query<(&Health, &Attack, &Defense)>) {
+    let mut enemies: Vec<_> = enemies.iter().collect();
```

Comment:
```suggestion
    // Oh no! An allocation!
    let mut enemies: Vec<_> = enemies.iter().collect();
```

Reply:
Added! Without the oh no, since you can't avoid allocation with sorts :)